travis/run_pylint.py

```diff
@@ -168,7 +168,7 @@ def pylint_run(is_pr, version, dir):
     real_errors = main(cmd, standalone_mode=False)
     res = dict(
         (key, value) for key, value in real_errors['by_msg'].items()
-        if key not in beta_msgs)
+        if key not in beta_msgs) if 'by_msg' in real_errors else dict()
```

Comment:
I think it's easier and more readable this way:
```python
    res = dict(
         (key, value) for key, value in real_errors.get('by_msg', {}).items()
        if key not in beta_msgs)
```

Reply:
beteer use (real_error.get(‘by_msg’) or {}) incorder to avoid the same error if the key is defined with None value

Reply:
The problem is the <code>.items()</code>. You can't do <code>{}.items()</code>

Reply:
Of course you can do it. It returns `[]`.

Reply:
Done!